style.css

```diff
@@ -0,0 +1,399 @@
+@media (min-width: 768px) {
```

Comment:
- [ ] Please note that it is part of the project requirements that your mobile and desktop version should be one web app. This is because every web app in the real world is one application that is responsive over different screen sizes. I will suggest you copy the styling in your mobile app here and paste it outside the media query, then you fix any little errors you might get on both the mobile and desktop screen sizes.

### Your mobile version
![1](https://user-images.githubusercontent.com/94551484/213780425-6eb694bc-e0b5-4949-91a2-7cb0a72f6ad6.png)